tests/unit/src/main/java/org/mule/tck/SimpleUnitTestSupportLifecycleSchedulerDecorator.java

```diff
@@ -87,16 +88,12 @@ public void execute(Runnable command) {
 
   @Override
   public ScheduledFuture<?> scheduleWithCronExpression(Runnable command, String cronExpression) {
-    ScheduledFuture<?> recurrentTask = decorated.scheduleWithCronExpression(wrap(command), cronExpression);
-    recurrentTasks.add(recurrentTask);
```

Comment:
aren't you breaking the assertions around tasks being completed and properly disposed and other shuutdown validations with this? My feat is that we reduce execution time at the expense of no longer catching potential leaks and other herbs

Reply:
no, the only code where objects of this class are being built mocks these methods, so these methods were never actually being executed.